.github/workflows/example-run-report.yml

```
@@ -96,8 +96,20 @@ jobs:
           else
             echo "result=false" >> $GITHUB_OUTPUT
           fi
+      - name: "Check if last comment is already from actions"
+        id: check-last-comment
+        env:
+          PR: ${{ needs.make-macos-screenshots-available.outputs.pr-number }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          if [[ `gh api --jq '.[-1].user.login' /repos/bevyengine/bevy/issues/18139/comments` == 'github-actions[bot' ]]
```

```suggestion
          if [[ `gh api --jq '.[-1].user.login' /repos/bevyengine/bevy/issues/$PR/comments` == 'github-actions[bot' ]]
```

Shouldn't this use the PR env var too?

It might be good to put this into an /actions folder for better reusability.

---

oh 🤦 a copy paste I missed to fix. thanks!